src/decorator/decorator-call-expr-identifier-reference.case

```diff
@@ -0,0 +1,44 @@
+// Copyright (C) 2022 Chengzhong Wu. All rights reserved.
+// This code is governed by the BSD license found in the LICENSE file.
+
+/*---
+desc: >
+  Decorator @ DecoratorCallExpression
+info: |
+  DecoratorCallExpression[Yield, Await] :
+    DecoratorMemberExpression[?Yield, ?Await] Arguments[?Yield, ?Await]
+
+  DecoratorMemberExpression[Yield, Await] :
+    IdentifierReference[?Yield, ?Await]
+    PrivateIdentifier
+    DecoratorMemberExpression[?Yield, ?Await] . IdentifierName
+
+  IdentifierReference[Yield, Await] :
+    Identifier
+    [~Yield] yield
+    [~Await] await
+
+template: syntax/valid
+---*/
+
+//- setup
+function decorator() {
+  return () => {};
+}
+var _ = decorator;
+var \u{6F} = decorator;
+var \u2118 = decorator;
+var ZW_\u200C_NJ = decorator;
+var ZW_\u200D_J = decorator;
+var yield = decorator;
+var await = decorator;
+
+//- decorators
+@$()
```

Comment:
This test seems to be incorrect because it does not define this (`$()`) decorator.

Reply:
Thank you for raising these! I've submitted https://github.com/tc39/test262/pull/3585 to fix these.